exampleCourse/questions/examplesRGetData/server.py

```diff
@@ -0,0 +1,62 @@
+import io
+import numpy
+import random
+import rpy2.robjects as robjects
+from pandas import DataFrame
+
+def generate(data):
+    
+    # Get list of data sets in base R
+    # val = robjects.r('d = data(package = "datasets")$results[,"Item"]; d[!grepl("[[:space:]]", d)]')
```

Comment:
Why do we have commented-out code in this file? Is it illustrating something, or is it just left over? If the latter, maybe remove it?